front-end/studio/src/app/pages/apis/{apiId}/editor/_components/common/inline-editor.base.ts

```diff
@@ -261,7 +261,7 @@ export abstract class TextAreaEditorComponent extends TextInputEditorComponent {
 
     /**
      * Subclasses can override this to provide validation status of the current value.
-     * @return {boolean}
```

Comment:
Lots of jsdoc changes in this diff.  I think WebStorm adds this automatically.  I'm happy to remove them all but am wondering why. :)

Reply:
Yeah figured that one out.  WebStorm adds the {Type} automatically ATM, but the next patched version of it will add an option to toggle that behavior (and set the default for typescript to false).  So that's good. :)

I'll go through all of Apicurio at some point and remove them.

Reply:
You could do `find . -name '*.ts' -exec sed -i -e 's/\(@param\|@return\) {[^}]*}/\1/' {} \;` or some other bulk magic.

Reply:
+1 - I do that sort of thing with regular expressions from within Eclipse.  It has nice support for find/replace across files using regexp and with full preview before approving the change.

Reply:
FWIW: https://youtrack.jetbrains.com/issue/WEB-28964